pkg/acceptance/bettertestspoc/README.md

```diff
@@ -0,0 +1,460 @@
+        	            	object WAREHOUSE["VKSENEIT_535F314F_6549_348F_370E_AB430EE4BC7B"] assertion [5/13]: failed with error: expected max cluster count: 12; got: 8
+        	            	object WAREHOUSE["VKSENEIT_535F314F_6549_348F_370E_AB430EE4BC7B"] assertion [6/13]: failed with error: expected min cluster count: 13; got: 2
+        	            	object WAREHOUSE["VKSENEIT_535F314F_6549_348F_370E_AB430EE4BC7B"] assertion [7/13]: failed with error: expected type: STANDARD; got: ECONOMY
+        	            	object WAREHOUSE["VKSENEIT_535F314F_6549_348F_370E_AB430EE4BC7B"] assertion [8/13]: failed with error: expected auto suspend: 123; got: 1000
+        	            	object WAREHOUSE["VKSENEIT_535F314F_6549_348F_370E_AB430EE4BC7B"] assertion [9/13]: failed with error: expected auto resume: false; got: true
+        	            	object WAREHOUSE["VKSENEIT_535F314F_6549_348F_370E_AB430EE4BC7B"] assertion [10/13]: failed with error: expected resource monitor: some-id; got: OOUJMDIT_535F314F_6549_348F_370E_AB430EE4BC7B
+        	            	object WAREHOUSE["VKSENEIT_535F314F_6549_348F_370E_AB430EE4BC7B"] assertion [11/13]: failed with error: expected comment: bad comment; got: comment
+        	            	object WAREHOUSE["VKSENEIT_535F314F_6549_348F_370E_AB430EE4BC7B"] assertion [12/13]: failed with error: expected enable query acceleration: false; got: true
+        	            	object WAREHOUSE["VKSENEIT_535F314F_6549_348F_370E_AB430EE4BC7B"] assertion [13/13]: failed with error: expected query acceleration max scale factor: 12; got: 90
+        	Test:       	TestInt_Warehouses/create:_complete
+```
+
+## Adding new resource assertions
+For object `abc` create the following files with the described content in the `assert` package:
+- `abc_resource.go`
+```go
+type AbcResourceAssert struct {
+    *ResourceAssert
+}
+
+func AbcResource(t *testing.T, name string) *AbcResourceAssert {
+    t.Helper()
+
+    return &AbcResourceAssert{
+        ResourceAssert: NewResourceAssert(name, "resource"),
+    }
+}
+
+func ImportedAbcResource(t *testing.T, id string) *AbcResourceAssert {
+    t.Helper()
+
+    return &AbcResourceAssert{
+        ResourceAssert: NewImportedResourceAssert(id, "imported resource"),
+    }
+}
+```
+Two methods for each parameter (let's say parameter name is xyz):
+```go
+func (w *AbcResourceAssert) HasXyz(expected string) *AbcResourceAssert {
+    w.assertions = append(w.assertions, valueSet("xyz", expected))
+    return w
+}
+
+func (w *AbcResourceAssert) HasNoXyz() *AbcResourceAssert {
+    w.assertions = append(w.assertions, valueNotSet("xyz"))
+    return w
+}
+```
+
+- `abc_show_output.go`
+```go
+type AbcShowOutputAssert struct {
+    *ResourceAssert
+}
+
+func AbcShowOutput(t *testing.T, name string) *AbcShowOutputAssert {
+    t.Helper()
+    w := AbcShowOutputAssert{
+        NewResourceAssert(name, "show_output"),
+    }
+    w.assertions = append(w.assertions, valueSet("show_output.#", "1"))
```

Comment:
Shouldn't this be a separate assert or the implicit assert should have a different building method? For example, If I would like to assert that `show_output` is empty I should be able to do `AbcShowOutput(...).IsEmpty()`, but with implicit assert it would fail anyway, right? The implicit assert could be added in `AbcNonEmptyShowOutput(...).Has...()`, wdyt (same in parameters)

Reply:
Left this implementation because there is currently no use case we could think of where showoutput should be empty for the resource. Added two entries to readme referencing this topic.